packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Providers/TypeProvider.cs

```diff
@@ -361,10 +361,42 @@ private bool ShouldGenerate(PropertyProvider property)
                 }
             }
 
-            if (customPropertyReplacements.TryGetValue(property.Name, out PropertyProvider? customProp))
+            var customPropertySerializationNames = new Dictionary<string, string>();
```

Comment:
Do we need this dictionary? Could we instead skip attributes where the property name doesn't match either property.Name or a value in customPropertyReplacements?